domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleClaim.java

```diff
@@ -66,10 +66,11 @@ public static Claim getWithDefaultResponse() {
         return getWithResponse(SampleResponse.validDefaults());
     }
 
-    public static Claim getWithResponse(final Response response) {
+    public static Claim getWithResponse(final Response responseData) {
```

Comment:
this looks like a merge that didn't go right, can you undo these changes please?
REQ